src/pages/community/how_to_contribute.md

```diff
@@ -92,7 +92,7 @@ Once the workflow has run and has given a pass (a +1 vote) a committer will revi
 As part of the Kubernetes shim there is also a set of tests defined called the _end to end_ tests.
 These tests are more complex to execute as they require some extra tools to be installed.
 Execution, and a pass, of the tests is strongly recommended for changes to the Kubernetes shim repository.
-They can be executed locally via `make e2e_test`.
+They can be executed locally via `run-e2e-tests.sh`.
```

Comment:
It seems to me the location of the script is not straightforward. How about yunikorn-k8shim/scripts/run-e2e-tests.sh?

Reply:
Thanks for the comment !